translations/binaries/slps_strings_map_location.json

```
@@ -0,0 +1,851 @@
+            "available_bytes_size": 16,
+            "translation": "세인갈드 성A[END]"
+        },
+        {
+            "offset": 1730064,
+            "string": "ハーメンツA(夜)[END]",
+            "original_bytes_length": 16,
+            "available_bytes_size": 16,
+            "translation": "하멘츠A(밤)[END]"
+        },
+        {
+            "offset": 1730080,
+            "string": "ハーメンツA[END]",
+            "original_bytes_length": 12,
+            "available_bytes_size": 15,
+            "translation": "하멘츠A[END]"
+        },
+        {
+            "offset": 1730096,
+            "string": "神殿A(夜)[END]",
+            "original_bytes_length": 10,
+            "available_bytes_size": 15,
+            "translation": "신전A(밤)[END]"
+        },
+        {
+            "offset": 1730112,
+            "string": "神殿A[END]",
+            "translation": "신전A[END]",
+            "original_bytes_length": 6,
+            "available_bytes_size": 7
+        },
+        {
+            "offset": 1730120,
+            "string": "神殿へと続く洞窟A(夜)[END]",
+            "original_bytes_length": 22,
+            "available_bytes_size": 23,
+            "translation": "신전으로 이어지는 동굴A(밤)[END]"
+        },
+        {
+            "offset": 1730144,
+            "string": "神殿へと続く洞窟A[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23,
+            "translation": "신전으로 이어지는 동굴A[END]"
+        },
+        {
+            "offset": 1730168,
+            "string": "ジェノスA(夜)[END]",
+            "original_bytes_length": 14,
+            "available_bytes_size": 15,
+            "translation": "제노스A(밤)[END]"
+        },
+        {
+            "offset": 1730184,
+            "string": "ジェノスA[END]",
+            "original_bytes_length": 10,
+            "available_bytes_size": 15,
+            "translation": "제노스A[END]"
+        },
+        {
+            "offset": 1730200,
+            "string": "ファンダリアの山道B(夜)[END]",
+            "original_bytes_length": 24,
+            "available_bytes_size": 24,
+            "translation": "팬더리아의 산길B(밤)[END]"
+        },
+        {
+            "offset": 1730224,
+            "string": "ファンダリアの山道A(夜)[END]",
+            "original_bytes_length": 24,
+            "available_bytes_size": 24,
+            "translation": "팬더리아의 산길A(밤)[END]"
+        },
+        {
+            "offset": 1730248,
+            "string": "ファンダリアの山道B[END]",
+            "original_bytes_length": 20,
+            "available_bytes_size": 23,
+            "translation": "팬더리아의 산길B[END]"
+        },
+        {
+            "offset": 1730272,
+            "string": "ファンダリアの山道A[END]",
+            "original_bytes_length": 20,
+            "available_bytes_size": 23,
+            "translation": "팬더리아의 산길A[END]"
+        },
+        {
+            "offset": 1730296,
+            "string": "トーンの山小屋・裏山B(夜)[END]",
+            "original_bytes_length": 26,
+            "available_bytes_size": 31,
+            "translation": "톤의 산막・뒷산B(밤)[END]"
```

https://github.com/ToD-DC-Kor/kor-patch/pull/445#discussion_r1773371048 에 적어주신 의견에 따라 '톤의 산장'으로 표현되어야 할 것 같네요

```suggestion
            "translation": "톤의 산장・뒷산B(밤)[END]"
```